pkg/credentials/credentials.go

```diff
@@ -129,7 +195,7 @@ func AppendPluginSpecificSpecs(velero *oadpv1alpha1.Velero, veleroDeployment *ap
 			veleroDeployment.Spec.Template.Spec.InitContainers = append(
 				veleroDeployment.Spec.Template.Spec.InitContainers,
 				corev1.Container{
-					Image:                    pluginSpecificMap.PluginImage,
+					Image:                    getPluginImage(velero),
```

Comment:
I think this needs to be reworked. You are calling `getPluginImage()` on every loop and `getPluginImage()` is looping through each plugin, finding if the plugin exists, and returning the image name if the plugin is found.

If I have a Velero CR with `aws` and `gcp` plugins installed, `getPluginImage()` will always return the `aws` image because its the first plugin found.

Reply:
With this comment: https://github.com/openshift/oadp-operator/pull/243/files#r698782601

Now you could do:
```
getPluginImage(pluginSpecificMap.PluginName, velero)
```